src/components/policy/policy_external/include/policy/cache_manager.h

```diff
@@ -955,6 +958,7 @@ class CacheManager : public CacheManagerInterface {
   sync_primitives::Lock backuper_locker_;
   BackgroundBackuper* backuper_;
   const PolicySettings* settings_;
+  volatile bool is_policy_table_loaded_;
```

Comment:
@LitvinenkoIra Do we need `volatile` here , does this variable change from other thread?